agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -1,13 +1,18 @@
 import os
 import logging
 from typing import Any, Optional
-from agenta.client import client
 
 from .utils.globals import set_global
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
+from agenta.client.client import ClientWrapper
+
+client_wrapper = ClientWrapper()
```

Comment:
how would that work? This is the SDK and not the CLI. The SDK does not have a config.toml. And you do neither provide the api key nor the the host. That won't work.